### ISSUE TITLE
SKINTONE OF AFRO PUFF GIRL 

### ISSUE THREAD

**Issue by PriscilliaCBI** [collective-for-black-iranians/website#24]
![unnamed](https://user-images.githubusercontent.com/69783021/90360041-5e554800-e00f-11ea-9401-0ed1fc0afa57.jpg)
Noticed that there is a skin tone difference from the pdf file (left) that doesn't match the site (right). Spotted only on the ABOUT page, the girl at the bottom, sitting by the water alone (As you scroll down)

**Comment by pmarashi** [comment]
I just want to clarify @PriscilliaCBI that it's a PSD file :)

**Comment by pmarashi** [comment]
Hi @dphiffer ... @PriscilliaCBI is correct - I think the issue here is when I shared the gif file to show the animation, you used the gif file, which is actually compressed and does not show the correct skin colors. Can you please update and use the correct skin colors in the PSD files? This applies to all the animations.

**Comment by dphiffer** [comment]
> Would you prefer that we send you a different export?

Yes, please do. Color matching is tricky business and I'd feel better having you on it.

**Comment by pmarashi** [comment]
Great! I re-exported the files for this animation... https://www.dropbox.com/s/xg0hkrg0nunphr6/Boy_with_Kite_About-NEW.gif?dl=0
